packages/gestalt/src/Box.js

```diff
@@ -80,96 +54,818 @@ extra runtime typechecks in the transformers for performance.
+   */
+  maxHeight?: number | string,
+  /**
+   * Use numbers for pixels: maxWidth={100} and strings for percentages: maxWidth="100%". See the [sizing](https://gestalt.pinterest.systems/box#Sizing) variant for more info.
+   */
+  maxWidth?: number | string,
+  /**
+   * Use numbers for pixels: minHeight={100} and strings for percentages: minHeight="100%". See the [sizing](https://gestalt.pinterest.systems/box#Sizing) variant for more info.
+   */
+  minHeight?: number | string,
+  /**
+   * Use numbers for pixels: minWidth={100} and strings for percentages: minWidth="100%". See the [sizing](https://gestalt.pinterest.systems/box#Sizing) variant for more info.
+   */
+  minWidth?: number | string,
+  /**
+   * See the [opacity](https://gestalt.pinterest.systems/box#Opacity) variant for more info.
+   */
+  opacity?: 0 | 0.1 | 0.2 | 0.3 | 0.4 | 0.5 | 0.6 | 0.7 | 0.8 | 0.9 | 1,
+  /**
+   * See the [overflow](https://gestalt.pinterest.systems/box#Overflow) variant for more info.
+   * Default: 'visible'
+   */
+  overflow?: 'visible' | 'hidden' | 'scroll' | 'scrollX' | 'scrollY' | 'auto',
+  /**
+   * Supports 3 responsive breakpoints: sm, md, lg. Each sets the padding from that breakpoint and up. See the [responsive padding](https://gestalt.pinterest.systems/box#Responsive-padding) variant for more info.
+   *
+   * Also available in responsive sizes: `smPadding`, `mdPadding`, `lgPadding`
+   * Default: 0
+   */
+  padding?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  smPadding?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  mdPadding?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  lgPadding?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  /**
+   * Horizontal padding (left/right)
+   *
+   * Also available in responsive sizes: `smPaddingX`, `mdPaddingX`, `lgPaddingX`
+   * Default: 0
+   */
+  paddingX?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  smPaddingX?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  mdPaddingX?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  lgPaddingX?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  /**
+   * Vertical padding (top/bottom)
+   *
+   * Also available in responsive sizes: `smPaddingY`, `mdPaddingY`, `lgPaddingY`
+   * Default: 0
+   */
+  paddingY?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  smPaddingY?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  mdPaddingY?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  lgPaddingY?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,
+  /**
+   * See the [absolute positioning](https://gestalt.pinterest.systems/box#Absolute-positioning) variant for more info.
+   * Default: 'static'
+   */
+  position?: 'static' | 'absolute' | 'relative' | 'fixed',
+  /**
+   * Ref that is forwarded to the underlying input element. See the [using as a ref](https://gestalt.pinterest.systems/box#Using-as-a-ref) variant for more info.
+   */
+  ref?: UnionRefs,
```

Comment:
Interestingly, this type _is_ resolved correctly. It appears the line is at imported types.